docs/api/js/types.md

```diff
@@ -4,6 +4,7 @@ All the types which are used in the SDK are referenced here:
 
 ```typescript
 import { BytesLike, BigNumberish, providers, BigNumber } from 'ethers';
```

Comment:
Why not just provide a link to our sdk, once it will be public.

Reply:
@Deniallugo what do you mean? These objects can only be imported from `ethers`. `ethers` is a peer dependency for our SDK anyway

Reply:
If I understood you correctly, even once it will be public,  SDK should be documented, in ideally every function, type, etc. Now it is not public.

Reply:
The whole snippet looks redundant. 
You copy and paste the file from js SDK.

Reply:
it was before like this. It is not my feature...so I think we need to make changes for it in the separate PR

Reply:
But it's still weird to have such snippets in the documentation. if we have an opportunity to have todos in our docs it definitely deserves a todo

Reply:
agree.
will add now